torch_xla/core/xla_model.py

```diff
@@ -30,7 +30,15 @@ def parse_xla_device(device):
 
 
 def get_xla_supported_devices(devkind=None, max_devices=None):
-  """get_xla_supported_devices (PLACEHOLDER)"""
+  """Returns a list of supported devices of a given kind.
+
+  Args:
+    devkind: If specified, one of `TPU`, `GPU` or `CPU`.
```

Comment:
This indicates we support GPU as well :P Maybe we can add a warning about GPU backend?